lib/index.js

```diff
@@ -991,7 +1001,9 @@ module.exports = {
                     subject.ownerDocument.contentType
                   ),
                   'to satisfy',
-                  value.children
+                  Array.isArray(value.children)
+                    ? convertChildrenToSatisfySpec(value.children, isHtml)
+                    : value.children
```

Comment:
Shouldn't we always convertChildrenToSatisfySpec, what if I'm providing a DOM tree as a single items? Maybe I don't have the full overview 🤔

Reply:
I don't believe so - based on my understanding of the rest of the assertions, I believe it is possible for a DOMNodeList object to be supplied as the value of `children` and the comparison should still work. If individual DOM nodes are supplied they'd be in an array which would need the conversion :)